src/editor/dag.rs

```diff
@@ -702,6 +703,30 @@ impl<'arena, Node: Ast<'arena>> Dag<'arena, Node> {
         self.write_text(&mut s, format);
         s
     }
+
+    /// Convert the Dag to .dot codes, and print the codes to log
+    pub fn to_graph(&self) {
```

Comment:
I think that this should just return the string and let the caller deal with it - we can handle writing it to a file once command mode is implemented.